modules/backup-and-restore/pages/cross-cluster-backup.adoc

```diff
@@ -12,78 +12,96 @@ Your cluster is offline during the restore process.
 * You have access to the TigerGraph Linux user account on your cluster.
 All commands must be run from the TigerGraph Linux user.
 * You have access to a data backup from another cluster.
+* [xref:_procedure_v3_9_1_and_below[3.9.1 and earlier]] You have the metadata backup that goes with the data backup.
 * You have xref:configurations.adoc[configured backup and restore] on the new cluster.
 
-NOTE: Since v3.9.2, it is no longer required to create a backup for a backups metadata or upload the metadata to a new cluster.
-A backup’s metadata is now stored as a part of a backup's files, but the previous method is still supported. *See xref:_prerequisites_v3_9_1_and_below[].
-
-Simply run:
-
-[source,console,]
---
-gadmin backup restore <backup_tag>
---
+== Procedure
 
-Just like restoring a backup in its original cluster.
+=== 1) Copy Data Backup to a New Cluster
 
-== Copy data backup to new cluster
+==== For Amazon S3 Backup:
 
-For an Amazon S3 backup, the restore flow will try to find a S3 object:
+If your backups are stored in Amazon S3 buckets, you can skip this step as long as you configure:
```

Comment:
```suggestion
If your backups are stored in Amazon S3 buckets, set the following configuration parameter to be the location of your backup files.
```
I think your statement is technically correct, but it's phrased in a backward, exception-case way.
"You don't need to do anything, unless you do."  --> Just be direct and tell me what to do.

Reply:
Got it! Thank you. I will keep this in mind. Updated with latest commits.